RobotTesting/Assets/Materials/RobotandMarkerReplacement.shader

```diff
@@ -0,0 +1,142 @@
+Shader "Custom/RobotandMarkerReplacement"
+{
+   Properties
+   {
+ _MainTex ("Texture", 2D) = "white" {}
+   }
+
+   SubShader
+   {
+ Tags { "RenderType"="Opaque" "RenderQueue"="First"}
+
```

Comment:
For the mask image, can you set the background to black? I'm seeing some strange line at the bottom of the image with the gray but not with the black. This could also be an issue with my monitor, but better to be safe.